test/basic-querying.test.js

```diff
@@ -336,12 +350,15 @@ describe('basic-querying', function() {
       });
     });
 
+    var beatleNames4 = ['George Harrison', 'Ringo Starr', 'Pete Best', 'Stuart Sutcliffe'];
+
     it('should support number "gte" that is satisfied', function(done) {
-      User.find({order: 'seq', where: {order: {'gte': 3},
-      }}, function(err, users) {
+      User.find({order: 'seq', where: {order: {'gte': 3}}}, function(err, users) {
         should.not.exist(err);
         users.should.have.property('length', 4);
-        users[0].name.should.equal('George Harrison');
+        for (var ix = 0; ix < 4; ix++) {
```

Comment:
By default, LoopBack sort results from `find` using PK if no `order` clause is present in the filter object.